src/utils/validators.js

```diff
@@ -692,6 +701,82 @@ const validateTextcard = (cardError, field, value) => {
   return newHighlightError
 }
 
+const validateInfocolsSection = (sectionError, sectionType, field, value) => {
+  const newSectionError = sectionError
+  let errorMessage = ""
+  switch (field) {
+    case "title": {
+      // Title is too short
+      if (value.length <= INFOCOLS_BLOCK_TITLE_MIN_LENGTH) {
+        errorMessage = `Title cannot be empty.`
+      }
+      // Title is too long
+      if (value.length >= INFOCOLS_BLOCK_TITLE_MAX_LENGTH) {
+        errorMessage = `Title should be shorter than ${INFOCOLS_BLOCK_TITLE_MAX_LENGTH} characters.`
+      }
+      break
+    }
+    case "subtitle": {
+      // Subtitle is too long
+      if (value.length >= INFOCOLS_BLOCK_SUBTITLE_MAX_LENGTH) {
+        errorMessage = `Subtitle should be shorter than ${TEXTCARDS_BLOCK_SUBTITLE_MAX_LENGTH} characters.`
+      }
+      break
+    }
+    case "linktext": {
+      // Link text is too long
+      if (value.length >= INFOCOLS_BLOCK_LINKTEXT_MAX_LENGTH) {
+        errorMessage = `Text should be shorter than ${INFOCOLS_BLOCK_LINKTEXT_MAX_LENGTH} characters.`
+      }
+      break
+    }
+    case "url": {
+      // if (!permalinkRegexTest.test(value)) {
```

Comment:
hmm could we just do a lose validation in the meantime? hunch here is that a basic incomplete one > no validation

Reply:
Preferring to do this in a separate PR - none of the components in this file have url validation. should update all at once